client/docs/plugin_model.md

```diff
@@ -0,0 +1,169 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Building Contract Plugins #
+
+The plugins module provides a means to build and use operations that
+is independent (as much as possible) of the actual invocation of the
+operation. For example, the `mock-contract` plugin wraps the
+`inc_value` method in the contract object with an interface so that it
+can be invoked as part of a Python script, a pdo-shell script, or a
+bash script. Similarly, the `mint_tokens` command may invoke a series
+of operations across multiple contract objects in order to mint new
```

Comment:
suggest remove references to NFTs for now, until exchange repo is open sourced.

Reply:
I would suggest leaving this in and, when we push the exchange repo that we reference it from this document. There are no details on exchange contracts in this document, the contracts are really just "references" to abstract ideas.